src/node_errors.h

```diff
@@ -138,6 +139,7 @@ void OnFatalError(const char* location, const char* message);
   V(ERR_INVALID_TRANSFER_OBJECT, "Found invalid object in transferList")       \
   V(ERR_MEMORY_ALLOCATION_FAILED, "Failed to allocate memory")                 \
   V(ERR_OSSL_EVP_INVALID_DIGEST, "Invalid digest used")                        \
+  V(ERR_MESSAGE_FAILED, "postMessage() Failed")                                \
```

Comment:
This would need to be documented. However, I would be against adding an error code like this in general.

1. Most of the time, we use standard `DOMException`s for other situations in which `postMessage()` fails.
2. `ERR_MESSAGE_FAILED` does not provide any extra value as an error code. If we do want to use Node.js errors, we should create a different error code for each of the different conditions that lead to an error being thrown here (otherwise people just fall back to parsing the error message and we’re back at the beginning).